### PR TITLE
Updating the CONTRIBUTING file

### LINKED ISSUE #15: Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Anas-mhameed**
Adding a standard about PR Review Process
resolve #15

### CODE REVIEW

**Review by rawadabu — APPROVED** [review]

**Review by finziamit — APPROVED** [review]
LGTM

**Review by Shalevro2 — APPROVED** [review]

**Review by yaelnoyman1 — APPROVED** [review]

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: ```... before creating the new PR. todo```

Did you forget to delete it?

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: maintainers? 
Please replace the terms are Reviewers -> Contributors, and mentos -> Core Reviewers

**Review by Yarboa — CHANGES_REQUESTED** [review]
Consider Renaming the ## Pre-Requirements -> Dev environment setup

Consider adding guideline for quick CR(Core Reviewer) Review:

Commit description should include subject, empty line followed with comment in more details
In case commit resolves and Issue please resolves #Issue_id

All automation test should pass successfully check (flake8 is just one test from more to come)

PR should be as smaller as possible.

PR should not have merge conflicts (rebase, push --force)

PR should (optional) include tests which covers the PR change

At least 2 Reviewers should Approve this PR

I hopw it makes more sense now

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: %s/mentors/Core Reviewers/

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: Consider markdown bold option
please refer this
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: same as above

**Review by Yarboa — APPROVED** [review]

**Review by Yarboa — CHANGES_REQUESTED** [review]
Please go over latest comments

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: Can explain, please ??

Comment (Yarboa) [reply CONTRIBUTING.md]: Please change mentors -> Core Reviewers (CR)

Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md:19]: Please consider this

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: Please note the difference between Reviewers and Core Reviewers

Consider this suggestion
5. Once PR gets enough approvals from the Reviewers, 
It is ready to merge,
6. Merge is done by Core Reviewers once it is approved by Reviewers.

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: I think you can remove this one

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: I think you can remove this one also, it is covered in next section

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: Please consider this:

Guidelines for quick PR merge

File: CONTRIBUTING.md
Comment (Yarboa) [inline CONTRIBUTING.md]: I think it is redundant

**Review by Yarboa — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md
Comment (Anas-mhameed) [reply CONTRIBUTING.md]: done

**Review by rawadabu — APPROVED** [review]
LGTM

**Review by Shalevro2 — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]

**Pull request merged**
